portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,14 @@
 #greeting-container {
   margin-top: 20px;
 }
+
```

Comment:
Is this CSS used anywhere?

Reply:
No, it's left over from the original template code. I'll delete it